templates/explore/repo_list.tmpl

```diff
@@ -29,7 +29,7 @@
 				</div>
 			</div>
 			<div class="description">
-				{{if .DescriptionHTML}}<p class="has-emoji">{{.DescriptionHTML}}</p>{{end}}
+				{{if .DescriptionHTML}}<p class=>{{.DescriptionHTML}}</p>{{end}}
```

Comment:
```suggestion
				{{if .DescriptionHTML}}<p>{{.DescriptionHTML}}</p>{{end}}
```
why not remove it completly ?

Reply:
thanks fixed -- wish there was a linter for templates!